evap/results/templates/result_bar.html

```diff
@@ -6,15 +6,19 @@
     <div data-toggle="tooltip" data-placement="left"
         title="{% if questionnaire_warning or result.warning %}{% trans "Only a few participants answered this question." %}</br></br>{% endif %}
         {% for answer, count in result.counts.items %}
-            {% if result.question.is_likert_question %}{{ answer|likertname }}{% else %}{{ answer }}{% endif %}: {{ count }}/{{ result.total_count }} ({{ count|percentage_one_decimal:result.total_count }}){% if not forloop.last %}</br>{% endif %}
+            {% if result.question.is_likert_question %}{{ answer|likertname }}{% elif result.question.is_positive_yes_no_question %}{{ answer|positiveyesnoname }}{% elif result.question.is_negative_yes_no_question %}{{ answer|negativeyesnoname }}{% else %}{{ answer }}{% endif %}: {{ count }}/{{ result.total_count }} ({{ count|percentage_one_decimal:result.total_count }}){% if not forloop.last %}</br>{% endif %}
```

Comment:
can you replace that if/elif with a new templatetag like this `{{ result.question|get_answer_name:answer }}` that handles all four answer types?